Project/demo/src/main/java/com/example/demo/model/Answer.java

```diff
@@ -0,0 +1,113 @@
+package com.example.demo.model;
+
+import org.springframework.data.annotation.CreatedDate;
+import org.springframework.data.jpa.domain.support.AuditingEntityListener;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.EntityListeners;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.Table;
+
+import java.time.Instant;
+
+@Entity
+@Table(name = "answer")
+@EntityListeners(AuditingEntityListener.class)
+public class Answer
+{
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @Column(nullable = false, length = 300)
```

Comment:
Why do we need to add answer length ? What if answer is longer than 300 characters ?